README.md

```diff
@@ -19,9 +19,9 @@ Project Bonneville was research aimed at determining best approaches to enabling
 
 ## Installing
 
-Once built, the result can be installed with the following command. This is just an example - see the install help output for all options, and a slightly more indepth example [here](usage.md):
+Once built, the result can be installed with the following command. This is just an example - see the vic-machine help output for all options, and a slightly more indepth example [here](usage.md):
 ```
-install.sh -g -t '<user>:<password>@<target-host>' -i <datastore-name> <vch-name>
+vic-machine -target target-host -iStore <datastore-name> -name <vch-name> -user <user> -passwd <password> -pool <resource pool path in govc format> -gen
```

Comment:
I'm not sure,  but short flag is not enough for all parameters, so I changed to this kind of longer flag.
For example, we have password, and also have pool flags, simply using the first character is not good, but with other characters might not be that easy to understand.
But anyway, I'm open for the naming suggestion.